src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -888,3 +888,85 @@ def regenerate_service_hooks_for_installation(
                 events=events,
                 url=webhook_url,
             )
+
+
+@instrumented_task(
+    name="sentry.sentry_apps.tasks.sentry_apps.broadcast_webhooks_for_organization",
+    taskworker_config=TaskworkerConfig(
+        namespace=sentryapp_tasks,
+        retry=Retry(
+            times=3,
+            delay=60 * 5,
+        ),
+        compression_type=CompressionType.ZSTD,
+        processing_deadline_duration=30,
+    ),
+    **TASK_OPTIONS,
+)
+@retry_decorator
```

Comment:
If we already have a resource change decorator on `send_resource_change_webhook` task, does it make sense to have one here too? We might double notify if a single installation is broken.